bigtable/writes/write_batch.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package writes
+
+// [START bigtable_writes_batch]
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/bigtable"
+)
+
+func writeBatch(w io.Writer, projectID, instanceID string, tableName string) error {
+	// projectID := "my-project-id"
+	// instanceID := "my-instance-id"
+	// tableName := "mobile-time-series"
+
+	ctx := context.Background()
+	client, err := bigtable.NewClient(ctx, projectID, instanceID)
+	if err != nil {
+		return fmt.Errorf("bigtable.NewAdminClient: %v", err)
+	}
+	tbl := client.Open(tableName)
+	columnFamilyName := "stats_summary"
+	timestamp := bigtable.Now()
+
+	muts := make([]*bigtable.Mutation, 2)
```

Comment:
It's uncommon to predeclare the size of the slice. It's more common to declare an uninitialized slice and add to it with `append`.

```suggestion
	var muts []*bigtable.Mutation
```

Then, below, you can add to it with:

```
muts := append(muts, ...)
```

Reply:
This gives me type errors when I try this. Maybe you can take a look with me. I'm still a go newb